.github/workflows/pull_request_integration_tests.yml

```diff
@@ -9,7 +9,7 @@ on:
 jobs:
   test:
     name: test
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Actually I think we need to pick something like RedHat, Ubuntu tends to update their kernels and they are up to 5.4 on 20.04 it seems.

Reply:
Maybe if it's available `redhat/ubi8:8.10-1054`?

Reply:
Yeah, that was just an experiment. In theory, kernel 5.4 does not support yet ringbuffers. Even that, most tests passed!

Reply:
Heh, yeah I wonder how did that work :)